lib/awful/mouse/init.lua

```diff
@@ -319,6 +318,32 @@ function mouse.object.get_current_widget()
     end
 end
 
+function mouse.object.set_current_widget() end
```

Comment:
Empty setters are no longer necessary since cdf16d4660d73c7db988723aaee40076f7eaa9fe. Please remove (all of them).